### PR TITLE
Disabling job to add items to v1.27 enhancements board

### PR DESCRIPTION
**Pull request by marosset**
/assign @salaxander

### CODE REVIEW

File: config/jobs/kubernetes/sig-k8s-infra/trusted/sig-release-release-team-jobs/release-team-periodics.yaml [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
-      - ./release-team/hack/sync-enhancements-github-project-beta.sh
-      env:
-        - name: GITHUB_PROJECT_BETA_NUMBER
-          value: "117"
-        - name: GITHUB_TOKEN
-          valueFrom:
-            secretKeyRef:
-              name: k8s-release-enhancements-triage-github-token
-              key: token
+# Re-enable once v1.28 project board is created (and update the project board number below)
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-k8s-infra/trusted/sig-release-release-team-jobs/release-team-periodics.yaml:2]: Suggestion: you can make a much smaller diff with just changing the interval field to an impossible cron date

Comment (BenTheElder) [reply config/jobs/kubernetes/sig-k8s-infra/trusted/sig-release-release-team-jobs/release-team-periodics.yaml:2]: that means things like dashboards won't go anywhere, and the revert PR is a quicker review

Comment (marosset) [reply config/jobs/kubernetes/sig-k8s-infra/trusted/sig-release-release-team-jobs/release-team-periodics.yaml:5]: done

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**